components/webui/server/.env

```
@@ -3,7 +3,11 @@
 # Reference: https://nodejs.org/en/learn/getting-started/nodejs-the-difference-between-development-and-production
 NODE_ENV=production
 
+# Database
 HOST=localhost
 PORT=3000
 CLP_DB_USER=clp-user
```

dotenv-linter (3.3.0)

[warning] [UnorderedKey] The CLP_DB_USER key should go before the HOST key

(UnorderedKey)